pytest.ini

```diff
@@ -12,6 +12,75 @@ markers =
+  once:'index_together' is deprecated. Use 'Meta.indexes' in 'main.\w+' instead.:django.utils.deprecation.RemovedInDjango51Warning:django.db.models.options
+
+  # FIXME: Update `awx.main.migrations._dab_rbac` and delete this entry.
+  # once:Using QuerySet.iterator.. after prefetch_related.. without specifying chunk_size is deprecated.:django.utils.deprecation.RemovedInDjango50Warning:django.db.models.query
+  once:Using QuerySet.iterator.. after prefetch_related.. without specifying chunk_size is deprecated.:django.utils.deprecation.RemovedInDjango50Warning:awx.main.migrations._dab_rbac
+
+  # FIXME: Delete this entry once the **broken** always-true assertions in the
+  # FIXME: following tests are fixed:
+  # * `awx/main/tests/unit/utils/test_common.py::TestHostnameRegexValidator::test_good_call`
+  # * `awx/main/tests/unit/utils/test_common.py::TestHostnameRegexValidator::test_bad_call_with_inverse`
+  once:assertion is always true, perhaps remove parentheses\?:pytest.PytestAssertRewriteWarning:
+
+  # FIXME: Figure this out, fix and then delete the entry. It's not entirely
+  # FIXME: clear what emits it and where.
+  once:Pagination may yield inconsistent results with an unordered object_list. .class 'awx.main.models.workflow.WorkflowJobTemplateNode'. QuerySet.:django.core.paginator.UnorderedObjectListWarning:django.core.paginator
+
+  # FIXME: Figure this out, fix and then delete the entry.
+  once::django.core.paginator.UnorderedObjectListWarning:rest_framework.pagination
+
+  # FIXME: Replace use of `distro.linux_distribution()` via a context manager
+  # FIXME: in `awx/main/analytics/collectors.py` and then delete the entry.
+  once:distro.linux_distribution.. is deprecated. It should only be used as a compatibility shim with Python's platform.linux_distribution... Please use distro.id.., distro.version.. and distro.name.. instead.:DeprecationWarning:awx.main.analytics.collectors
+
+  # FIXME: Figure this out, fix and then delete the entry.
+  once:\nUsing ProtocolTypeRouter without an explicit "http" key is deprecated.\nGiven that you have not passed the "http" you likely should use Django's\nget_asgi_application...:DeprecationWarning:awx.main.routing
+
+  # FIXME: Figure this out, fix and then delete the entry.
+  once:Channel's inbuilt http protocol AsgiHandler is deprecated. Use Django's get_asgi_application.. instead.:DeprecationWarning:channels.routing
+
+  # FIXME: Use `codecs.open()` via a context manager
+  # FIXME: in `awx/main/utils/ansible.py` to close hanging file descriptors
+  # FIXME: and then delete the entry.
+  once:unclosed file <_io.BufferedReader name='[^']+'>:ResourceWarning:awx.main.utils.ansible
+
+  # FIXME: Use `open()` via a context manager
+  # FIXME: in `awx/main/tests/unit/test_tasks.py` to close hanging file
+  # FIXME: descriptors and then delete the entry.
+  once:unclosed file <_io.TextIOWrapper name='[^']+' mode='r' encoding='UTF-8'>:ResourceWarning:awx.main.tests.unit.test_tasks
```

Comment:
so this line can be removed with @AlanCoding 's commit?

Reply:
@fosterseth I think he removed one entry already, I'm not sure about others. There were several different warnings in some files. Somebody needs to check it.